L74-java-channel-creds.md

```diff
@@ -434,11 +434,11 @@ server = ServerBuilder.forPort(443)
 server = ServerBuilder.forPort(8080)
     .build().start();
 // the user would now:
-ServerCredentials tlsCreds = new TlsServerCredentials(
+ServerCredentials tlsCreds = TlsServerCredentials.create(
     new File("cert.pem"), new File("cert.key"));
 server = Grpc.newServerBuilderForPort(443, tlsCreds)
     .build().start();
-server = Grpc.newServerBuilderForPort(8080, new InsecureChannelCredentials())
+server = Grpc.newServerBuilderForPort(8080, InsecureChannelCredentials.create())
```

Comment:
Shouldn't this be `InsecureServerCredentials`? The previous version had the same issue.

Reply:
Good eye. Fixed.